skel/handlers/index.go

```diff
@@ -106,7 +106,13 @@ func Setup(app *sonic.Runtime) {
 	go daemons.LicenseRenewal("finex", app, vaultService)
 
 	// Run FetchMarkets
-	go daemons.FetchMarkets(peatioClient, vaultService, opendaxConfig.Addr)
+	enabled, err := daemons.GetXLNEnabledFromVault(vaultService)
+	if err != nil {
+		log.Printf("Can't create peatio client: " + err.Error())
```

Comment:
I'm not sure about this error message, should be something about getting the xln_enabled value

Reply:
@vshatravenko what would you suggest for this? should we just ignore the error message? I mean, we're sort of just using the boolean value, while by default it always `false` when error occured.

Reply:
We need the error message, just it shouldn't mention peatio since it's about XLN